django/db/backends/mysql/features.py

```diff
@@ -47,11 +47,18 @@ class DatabaseFeatures(BaseDatabaseFeatures):
 
     supports_order_by_nulls_modifier = False
     order_by_nulls_first = True
-    test_collations = {
-        'ci': 'utf8_general_ci',
-        'non_default': 'utf8_esperanto_ci',
-        'swedish_ci': 'utf8_swedish_ci',
-    }
+
+    @cached_property
+    def test_collations(self):
+        character_set = 'utf8'
```

Comment:
Could use `charset` over `character_set`? It's a well-known "word" and losing the underscore might make this less noisy visually, especially in the f-strings.

Only a suggestion, feel free to ignore.

Reply:
Updated.